README.md

```diff
@@ -25,6 +25,8 @@ Plotly Charts and Plotly Chart Editor integration with Volto
   'visualization'. `eea.restapi` has a behavior to implement such a content
   type.
 
+![Plotlycharts](./docs/volto-plotlycharts.gif)
```

Comment:
@ana-oprea Please use the absolute path in order for this to work also on npm

Reply:
I added the absolute path (master branch). https://raw.githubusercontent.com/eea/volto-plotlycharts/master/docs/volto-plotlycharts.gif
Will work as expected when the gif file will be merge into master.